src/languages/en.js

```diff
@@ -98,6 +98,7 @@ export default {
             characterLimit: ({limit}) => `Exceeds the maximum length of ${limit} characters`,
             dateInvalid: 'Please enter a valid date',
             invalidCharacter: 'Invalid character',
+            invalidEmail: 'Invalid email',
```

Comment:
@fedirjh just wondering, why do you prefer having the error message inside `messages` instead of here? I feel like here makes more sense since this is a generic set of errors

Reply:
@fedirjh Do you want the `invalidEmail` message similar to `errorMessageInvalidPhone` inside `messages` or leave it as it is?

Reply:
I would put it inside `messages` because it's more of an informative message rather than an error message, but it's not a big deal either way.